### PR TITLE
[MIG][12.0] web_external_help.

### PR DESCRIPTION
**Pull request by hpatelserpentcs**
Added 
> web_external_help

### CODE REVIEW

File: web_external_help/README.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Model and Fields Help
+===========================
```
Comment (max3903) [inline web_external_help/README.rst]:
```suggestion
=================
Model and Fields Help
=================
```

File: web_external_help/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 - TODAY Serpent Consulting Services Pvt. Ltd.
+# (<http://www.serpentcs.com>)
```
Comment (max3903) [inline web_external_help/__manifest__.py]:
```suggestion
# Copyright (C) 2019 Serpent Consulting Services Pvt. Ltd.
```

File: web_external_help/models/fields_external_help.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        string="External Url")
+    external_help_id = fields.Many2one(
+        string="External Help",
+        comodel_name="fields.external.help",
+        ondelete="cascade")
+    model_id = fields.Many2one(
+        related='external_help_id.model_id', readonly=True)
+    model = fields.Char(related='model_id.model', readonly=True)
+    link_text = fields.Char(string="Link Text")
+    help_text = fields.Html(string="Help Text")
```
Comment (max3903) [inline web_external_help/models/fields_external_help.py]: @hpatelserpentcs Please separate this file to have one file per model.

File: web_external_help/security/res_groups.xml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (max3903) [inline web_external_help/security/res_groups.xml]:
```suggestion
```

File: web_external_help/static/src/css/web-external-help.css [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+    margin-left: 5px;
+    background: none;
+    border: none;
+}
+
+.model_help {
+    color: #666666;
+    background: none;
+    border: none;
+}
```
Comment (max3903) [inline web_external_help/static/src/css/web-external-help.css]: @hpatelserpentcs Missing newline at the EOF.

File: web_external_help/views/debug_menu.xml [hunk 6]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (max3903) [inline web_external_help/views/debug_menu.xml]:
```suggestion
```

File: web_external_help/views/view_fields_external_help.xml [hunk 7]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (max3903) [inline web_external_help/views/view_fields_external_help.xml]:
```suggestion
```

File: web_external_help/views/view_model_external_help.xml [hunk 8]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (max3903) [inline web_external_help/views/view_model_external_help.xml]:
```suggestion
```

**Comment by hpatelserpentcs** [comment]
@max3903 

Updated the review points.

**Pull request closed without merging**